.env.example

```
@@ -71,4 +71,15 @@ OPENROUTER_API_KEY=""
 
 
 # Jupiter API KEY
-JUPITER_API_KEY=""
\ No newline at end of file
+JUPITER_API_KEY=""
+
+# Sumsub KYC
+SUMSUB_API_KEY=""
+SUMSUB_SECRET_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)